azurelinuxagent/common/conf.py

```diff
@@ -479,8 +479,8 @@ def get_autoupdate_enabled(conf=__conf__):
     return conf.get_switch("AutoUpdate.Enabled", True)
 
 
-def get_autoupdate_frequency(conf=__conf__):
-    return conf.get_int("Autoupdate.Frequency", 3600)
+def get_agentupdate_frequency(conf=__conf__):
+    return conf.get_int("AgentUpdate.Frequency", 3600)
```

Comment:
We should not rename these parameters, since customers may be using them

Reply:
I agree customer may have taken dependency, my intention is this frequency is shared between self and rsm update. So I thought auto update my confuse. Will revert it.